server/src/Game.java

```diff
@@ -0,0 +1,114 @@
+        playerTwo = playerTwoName;
+        turn = "blue";
+
+        for(int i=0; i<9; i++){
+            for(int j=0; j<7; j++){
+                board[i][j] = "__";
+            }
+        }
+
+        board[0][0] = "r7"; //red team lion (power 7)
+        board[0][6] = "r6"; //red team tiger (power 6)
+        board[1][1] = "r4"; //red team dog (power 4)
+        board[1][5] = "r2"; //red team cat (power 2)
+        board[2][0] = "r1"; //red team mouse (power 1)
+        board[2][2] = "r5"; //red team leopard (power 5)
+        board[2][4] = "r3"; //red team wolf (power 3)
+        board[2][6] = "r8"; //red team elephant (power 8)
+        board[6][0] = "b8"; //blue team elephant (power 8)
+        board[6][2] = "b3"; //blue team wolf (power 3)
+        board[6][4] = "b5"; //blue team leopard (power 5)
+        board[6][6] = "b1"; //blue team mouse (power 1)
+        board[7][1] = "b2"; //blue team cat (power 2)
+        board[7][5] = "b4"; //blue team dog (power 4)
+        board[8][0] = "b6"; //blue team tiger (power 6)
+        board[8][6] = "b7"; //blue team lion (power 7)
+
+    }
+
+    void printBoard(){
+        // prints out the board in a readable fashion
+        for(int i=0; i<9; i++) System.out.println(Arrays.toString(board[i]));
+        System.out.println("");
+    }
+
+    void sendInput(String player, int row, int col){
+        // send input to the game in the format (player sending move, column selected, row selected)
+        if(isTurn(player)){
+            if(move.selectedCol == col && move.selectedRow == row){ // if the second half of the move is the same as the first half (re-selecting the same tile)
+                move.selectedCol = -1;
+                move.selectedRow = -1;
+            }
+            else if(move.isFriendlyUnit(row, col)){ // if the player is selecting one of their own tiles
+                move.selectedCol = col;
+                move.selectedRow = row;
+            }
+            else if(move.validTiles[row][col].equals("*")) { // if row, col is a valid tile to move to
+                makeMove(row, col);
+            }
+            move.updateValidTiles();
+        }
+    }
+
+    private void makeMove(int row, int col){
+        // once a valid move has been constructed, make it.
+        board[row][col] = board[move.selectedRow][move.selectedCol];
+        board[move.selectedRow][move.selectedCol] = "__";
+
+        if(turn.equals("blue")) turn = "red";
+        else turn = "blue";
+
+        move.selectedRow = -1;
+        move.selectedCol = -1;
+        move.updateValidTiles();
+
+        checkIfWinner();
+
+        System.out.println("");
+        printBoard();
+    }
+
+    private boolean isTurn(String player){
+        // returns true if it is the turn of the player submitting the move, else returns false.
+        return (turn.equals("blue") && player.equals(playerOne)) || (turn.equals("red") && player.equals(playerTwo));
+    }
+
+    private void checkIfWinner(){
```

Comment:
We might want some of these methods to return an integer or flag of some sort, so the server knows when to make an update to the database without having to call the object again. Return -1 if no winner, or the player ID if winner, etc. We can talk about it more.